src/frontend/components/ViewersList/components/ViewersListTextButton/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Button, Text } from 'grommet';
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledTextButton = styled(Text)`
+  color: white;
+  font-family: 'Roboto-Bold';
+`;
+
+interface ViewersListTextButtonProps {
+  onClick: () => void;
+  text: string;
+}
+
+export const ViewersListTextButton = ({
+  onClick,
+  text,
+}: ViewersListTextButtonProps) => {
+  return (
+    <Button onClick={onClick} style={{ padding: '0px' }}>
+      <Box
+        background="blue-active"
+        height="20px"
+        justify="center"
+        pad={{ horizontal: '10px' }}
+        round="6px"
+      >
+        <StyledTextButton size="0.625rem">{text}</StyledTextButton>
+      </Box>
```

Comment:
you should use label button's prop if you have just some text in your button

Reply:
fix in commit : [7e6f05cd](https://github.com/openfun/marsha/pull/1443/commits/7e6f05cdad8aa7e7794ebac44061a786ebb404a4)